apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -655,22 +609,10 @@ function wpcom_site_has_global_styles_feature( $blog_id = 0 ) {
 		return true;
 	}
 
-	if ( wpcom_site_has_global_styles_in_personal_plan( $blog_id ) ) {
-		/*
-		 * Flag site so users of the treatment group with a Personal plan can always have access
-		 * to Global Styles, even if the experiment has finished without including Global Styles
-		 * in the Personal plan.
-		 */
-		$has_personal_plan = wpcom_site_has_personal_plan( $blog_id );
-		$note              = 'See https://wp.me/paYJgx-3yE';
-		if ( $has_personal_plan ) {
-			if ( ! wpcom_global_styles_has_blog_sticker( 'wpcom-global-styles-personal-plan', $blog_id ) ) {
-				add_blog_sticker( 'wpcom-global-styles-personal-plan', $note, null, $blog_id );
-			}
-		} else {
-			remove_blog_sticker( 'wpcom-global-styles-personal-plan', $note, null, $blog_id );
-		}
-		return $has_personal_plan;
+	// Users who bought a personal plan during the personal/premium experiment should
+	// retain access to Global Styles.
+	if ( has_blog_sticker( 'wpcom-global-styles-personal-plan', $blog_id ) ) {
+		return true;
 	}
```

Comment:
This should be true only if they have a personal plan. When the customer doesn't have a personal plan anymore we should remove the sticker.